### PR TITLE
Line select metadata

### PR DESCRIPTION
**Pull request by chinmaychahar**
Metadata for examples/plotting/line_select.py was outdated so I updated it acc. to the set template 

- [x] issues: #11765

### CODE REVIEW

File: examples/plotting/line_select.py [hunk 1]
```diff
@@ -1,5 +1,11 @@
-""" Example demonstrating the picking of line objects.
-
+""" Example demonstrating the picking of line objects -using customJS.
```
Comment (bryevdv) [inline examples/plotting/line_select.py]:
```suggestion
""" Example demonstrating line selection together with with customJS.
```

File: examples/plotting/line_select.py [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+It involves clicking on any of the plotted lines to select/ deselect the line.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.models.CustomJS, bokeh.models.TapTool
+    :refs: :ref:`ug_basic_lines_line_multi`,:ref:`ug_interaction_js_callbacks`
```
Comment (bryevdv) [inline examples/plotting/line_select.py]: errant ref anchor
```suggestion
    :refs: :ref:`ug_basic_lines_multi`, :ref:`ug_interaction_js_callbacks`
```

File: examples/plotting/line_select.py [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+    :keywords: line, taptool, customjS
```
Comment (bryevdv) [inline examples/plotting/line_select.py]: typo
```suggestion
    :keywords: line, taptool, customjs
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @chinmaychahar a few comments

**Comment by bryevdv** [comment]
@chinmaychahar it looks like #13019 was already just previously opened almost at the same time by @Azaya89 to for this file. I'd like for the two of you to discuss and let me know which PR to keep, perhaps you can collaborate and both add commits to one PR (by way of comment code suggestions).

**Comment by chinmaychahar** [comment]
Sure, I'd be happy to collaborate and discuss :D 

@Azaya89 thanks for the changes you suggested in your PR. I went through them and they were almost identical to mine so I don't mind going forward with either of the PRs. 

Let me know your thoughts and we can incorporate the changes suggested by @bryevdv in the PR too if you're okay with it. Cheers!

**Comment by Azaya89** [comment]
> Sure, I'd be happy to collaborate and discuss :D
> 
> @Azaya89 thanks for the changes you suggested in your PR. I went through them and they were almost identical to mine so I don't mind going forward with either of the PRs.
> 
> Let me know your thoughts and we can incorporate the changes suggested by @bryevdv in the PR too if you're okay with it. Cheers!

Oh yeah, I think 

> Sure, I'd be happy to collaborate and discuss :D
> 
> @Azaya89 thanks for the changes you suggested in your PR. I went through them and they were almost identical to mine so I don't mind going forward with either of the PRs.
> 
> Let me know your thoughts and we can incorporate the changes suggested by @bryevdv in the PR too if you're okay with it. Cheers!

Yeah, I think we can merge yours instead, since the changes suggested by @bryevdv is an improvement overall. I will go ahead and close mine now.

**Comment by chinmaychahar** [comment]
> Yeah, I think we can merge yours instead, since the changes suggested by @bryevdv is an improvement overall. I will go ahead and close mine now.

Okay, I'll be making the changes requested to this PR then. Let me know if you have comments to add. Thanks!

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @chinmaychahar and @Azaya89 !

**Pull request merged**